base_riskscore/code/run_cvsl_riskscore.R

```diff
@@ -36,20 +36,33 @@ source(here("code", "utils.R")) # get CV-AUC for all algs
 
 ############ SETUP INPUT #######################
 # Read in data file
-dat_cleaned <- read.csv(here::here("..", "data_clean", data_name))
-inputFile <- dat_cleaned
+inputFile <- read.csv(here::here("..", "data_clean", data_name))
 
 # Identify the risk demographic variable names that will be used to compute the risk score
-risk_vars <- c(
-  "MinorityInd", "EthnicityHispanic", "EthnicityNotreported", "EthnicityUnknown", 
-  "Black", "Asian", "NatAmer", "PacIsl", "WhiteNonHispanic", 
-  "Multiracial", "Other", 
-  "Notreported", "Unknown",
-  "HighRiskInd", "Sex", "Age", "BMI"
-)
-
 # Identify the endpoint variable
-endpoint <- "EventIndPrimaryD57"
+if(study_name_code == "MODERNA"){
```

Comment:
`study_name_code == COVE`, no?

Reply:
You are right, David. Will update to COVE. 👍